pkg/cluster/info.go

```diff
@@ -174,3 +175,20 @@ func (c *Info) OperatorNamespace() string {
 
 	return constants.OperatorNamespace
 }
+
+func (c *Info) HasBroker(namespace string) (bool, error) {
```

Comment:
The function name indicates it checks for existence but it also has the hidden side-effect of setting the `Broker` field which I don't think is proper. I think it's cleaner to return the `Broker` instance instead.

```suggestion
func (c *Info) GetBroker(namespace string) (*v1alpha1.Broker, bool, error) {
```